libs/shared/ofrep-core/src/test/handlers.ts

```diff
@@ -121,14 +121,20 @@ export const handlers = [
       }
 
       const scopeValue = new URL(info.request.url).searchParams.get('scope');
+      const key = info.params.key;
+      const reason = key.includes('null-value')
+        ? StandardResolutionReasons.DEFAULT
+        : requestBody.context?.targetingKey
+          ? StandardResolutionReasons.TARGETING_MATCH
+          : StandardResolutionReasons.STATIC;
+
+      const value = key.includes('null-value') ? null : true;
```

Comment:
Send back a `value=null` and `reason=DEFAULT`, this is my preferred proposal for implementing a "defer to code default" evaluation in flagd.

Reply:
Would we need to send a variant at all?